packages/js/src/lib/load-formbricks.ts

```diff
@@ -53,7 +53,15 @@ const functionsToProcess: { prop: string; args: unknown[] }[] = [];
 export const loadFormbricksToProxy = async (prop: string, ...args: unknown[]): Promise<void> => {
   // all of this should happen when not initialized:
   if (!isInitialized) {
-    if (prop === "init") {
+    // We need to still support init for backwards compatibility
+    // but we should log a warning that the init method is deprecated
+    if (prop === "setup" || prop === "init") {
+      if (prop === "init") {
+        console.warn(
+          "🧱 Formbricks - Warning: The init method is deprecated and will soon be removed. Please use setup instead."
+        );
+      }
+
```

Comment:
I think only one prop can be available in any version of the `js` package. So maybe we can remove the check for both the prop